exporter/jaegerexporter/exporter.go

```diff
@@ -71,7 +72,7 @@ func (s *protoGRPCSender) pushTraceData(
 
 	batches, err := jaegertranslator.InternalTracesToJaegerProto(td)
 	if err != nil {
-		return td.SpanCount(), consumererror.Permanent(err)
+		return td.SpanCount(), consumererror.Permanent(errors.Wrap(err, "failed to push trace data via Jaeger exporter"))
```

Comment:
IIRC more used is now `fmt.Errorf` within the codebase.

Reply:
Looks like people prefer the Wrap, we should converge to that :)

Reply:
I had `fmt.Errorf` but noticed that the code base seems to use `errors.Wrap`. I don't have a preference, to be honest :)

Reply:
I think we should use `fmt.Errorf` with `%w` as we've moved to Go 1.14 and the new errors package is available now. We can actually get rid of `pkg/errors` package.

Reply:
The nice thing about `errors.Wrap` is its type safe and you don't need to add `...: %w"` to every single `fmt.Errorf` call wrapping another error

Reply:
See example https://play.golang.org/p/t9GxkVpiIhA